dictmanager/importer.h

```diff
@@ -22,7 +22,7 @@
 #define FCITX_IMPORTER_H
 
 #include <QString>
-#include <fcitx-qt/fcitxqtconnection.h>
```

Comment:
I mean #include <fcitxqtconnection.h> is what it supposed to be. without <FcitxQt5...